hack/install-gox.sh

```diff
@@ -24,12 +24,10 @@ cleanup() {
 trap cleanup EXIT
 
 install_gox() {
-  gobin="$(go env GOPATH)/bin"
   cd "${scratch_dir}"
-  env GOPATH="${scratch_dir}" \
-    GO111MODULE=off \
-    GOBIN="${gobin}" \
```

Comment:
when you remove GOBIN, how do you ensure gox ends up in PATH?

Is the "$(go env GOPATH)/bin" (which is probably /home/travis/go/bin) added to PATH already?

Reply:
yes.